src/components/RecipesDisplay/RecipesDisplay.css

```
@@ -0,0 +1,15 @@
+.RecipesDisplay {
```

you should keep consistant naming with your css classes. I would stick with kebab case
```suggestion
.recipes-display {
```